src/test/java/testsuite/integration/container/IntegrationTestSuite.java

```diff
@@ -0,0 +1,13 @@
+package testsuite.integration.container;
+
+import org.junit.platform.suite.api.SelectClasses;
+import org.junit.platform.suite.api.Suite;
+
+@Suite
+@SelectClasses({
+    AuroraMysqlIntegrationTest.class,
+    AuroraMysqlFailoverIntegrationTest.class,
+    AuroraMysqlHardFailureIntegrationTest.class,
+    ReplicationFailoverIntegrationTest.class
+})
+public class IntegrationTestSuite {}
```

Comment:
What's the purpose of this class?

Reply:
To run the tests in a specific order. Running the tests with network failure first then running the tests that issue a reboot/failover on the instances.